src/CiviliansAccountsMicroservice/Civilians.Api/Program.cs

```diff
@@ -17,21 +16,20 @@
 
 var app = builder.Build();
 
-app.UseMiddleware<GlobalExceptionsHandler>();
-
 if (app.Environment.IsDevelopment())
 {
     await app.MigrateDatabase();
     await app.SeedUsers();
     app.UseSwagger();
     app.UseSwaggerUI();
 }
+else
```

Comment:
What about exception handler for dev enviroment?

Reply:
In dev env I prefer wider description for any exception, including stack trace
Exceptions handler gives only exception name and short description with a status code - it's not enough for debugging

Reply:
Okey